tools/accuracy_checker/accuracy_checker/preprocessor/brats_preprocessing.py

```diff
@@ -0,0 +1,169 @@
+        bbox[1] = bbox_max
+
+        image.data = image.data[:, bbox[0, 0]:bbox[1, 0], bbox[0, 1]:bbox[1, 1], bbox[0, 2]:bbox[1, 2]]
+
+        image.metadata['box'] = bbox
+
+        return image
+
+
+class NormalizeBrats(Preprocessor):
+    __provider__ = "normalize_brats"
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'masked': BoolField(optional=True, default=False,
+                                description='Does not apply normalization to zero values. '
+                                            'Applicable for brain tumor segmentation models'),
+            'cutoff': NumberField(optional=True,
+                                  description='Species range of values - [-cutoff, cutoff]'),
+            'shift_value': NumberField(optional=True, default=0, description='Specifies shift value'),
+            'normalize_value': NumberField(optional=True, default=1, description='Specifies normalize value')
+        })
+
+        return parameters
+
+    def configure(self):
+        self.masked = self.config.get('masked')
+        self.cutoff = self.config.get('cutoff', None)
+        self.shift_value = self.config.get('shift_value')
+        self.normalize_value = self.config.get('normalize_value')
+
+    def process(self, image, annotation_meta=None):
+        image.data = self.normalize_img(image.data)
+        return image
+
+    def normalize_img(self, image):
+        for channel in range(image.shape[0]):
+            img = image[channel, :, :, :].copy()
+            if self.masked:
+                mask = img > 0
+                image_masked = np.ma.masked_array(img, ~mask)
+                mean, std = np.mean(image_masked), np.std(image_masked)
+            else:
+                mean, std = np.mean(img), np.std(img)
+
+            img -= mean
+            img /= std
+
+            if self.cutoff:
+                img = np.clip(img, -self.cutoff, self.cutoff)
+            img += self.shift_value
+            img /= self.normalize_value
+            if self.masked:
+                img[~mask] = 0
+            image[channel, :, :, :] = img
+
+        return image
+
+
+class SwapModalitiesBrats(Preprocessor):
+    __provider__ = 'swap_modalities'
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'modality_order': ListField(value_type=int, validate_values=True,
+                                        description="Specifies order of modality according to model input")
+        })
+
+        return parameters
+
+    def configure(self):
+        self.modal_order = self.get_value_from_config('modality_order')
+        if len(self.modal_order) != 4:
+            raise ConfigError('{} supports only 4 modality, but found {}'
+                              .format(self.__provider__, len(self.modal_order)))
+        if max(self.modal_order) != 3 or min(self.modal_order) != 0:
```

Comment:
as I know you can specify min_value and max_value for list values inside configuration if you set value_type as NumberField